### DISCUSSION TITLE
Quick question how to include a commit/fix into a sub version release?

### DISCUSSION THREAD

**Discussion by itsnode** [SAP/fundamental-ngx#9327] [Q&A]
My recent PR https://github.com/SAP/fundamental-ngx/pull/9268 merged into main branch recently. Any one knows how to get this fix into 0.36.3 release. I am asking because our current project is using 0.36.0, no plan to upgrade to the next stable release(0.39.0) this year due to lots of dependency. So I am looking to sub version upgrade of v0.36.3.

Please let me know what is the procedure to get this done. thanks.

**Comment by droshev** [comment]
WE did it
